app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -421,6 +454,23 @@ class AdvancedFilter extends React.Component {
     );
   };
 
+  // Render relative date specific options
+  renderRelativeOptions = (current, index, value) => {
```

Comment:
not a big deal or even something we have to do but wondering if it would make sense to not include past/future options for certain fields (also something that can be considered/done in the future), ex: doesn't make sense to have a future enrolled/latest report/LDE/symptom onset date

Reply:
@holmesie what do you think?  I know that this is only the first iteration and more options will be added in the future (like `over 10 days ago`) so might be worth getting feedback on just this?